### PR TITLE
specify cmp addr

### PR DESCRIPTION
**Pull request by chengjoey**
#### What type of this PR

Add one of the following kinds:
/kind bug

#### What this PR does / why we need it:
specify cmp addr

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](paste your link here)

#### Specified Reviewers:

/assign @your-reviewer

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |   specify cmp addr           |
| 🇨🇳 中文    |   cmp指定自身地址注册到eventbox           |

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: erda.yml [hunk 1]
```diff
@@ -518,6 +518,7 @@ services:
       UC_CLIENT_SECRET: "secret"
       ERDA_NAMESPACE: "default"
       ERDA_HELM_CHART_VERSION: "1.1.0"
+      CMP_ADDR: "cmp:9027"
```
Comment (sfwn) [inline erda.yml:521]: Add env in client side. Or globally.

**Review by sfwn — CHANGES_REQUESTED** [review]

File: erda.yml (hunk 1 above)
Comment (chengjoey) [reply erda.yml]: cmp register self to eventbox
add it to global envs

**Review by sfwn — APPROVED** [review]

**Pull request merged**